Dockerfile

```diff
@@ -6,8 +6,18 @@ LABEL authors="Harshil Patel" \
 COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
 
+
+RUN mkdir /opt/scripts
+
+COPY ./bin/* /opt/scripts/
+
+RUN find /opt/scripts/ -type f -iname "*.py" -exec chmod +x {} \; && \
+    find /opt/scripts/ -type f -iname "*.r"   -exec chmod +x {} \; && \
+    find /opt/scripts/ -type f -iname "*2bed" -exec chmod +x {} \;
+
 # Add conda installation dir to PATH (instead of doing 'conda activate')
 ENV PATH /opt/conda/envs/nf-core-atacseq-1.1.0/bin:$PATH
+ENV PATH /opt/scripts:$PATH
 
 # Dump the details of the installed packages to a file for posterity
 RUN conda env export --name nf-core-atacseq-1.1.0 > nf-core-atacseq-1.1.0.yml
```

Comment:
Addition needed for running in environments where `launchDir` is either not available, non-writable (eg. when running on AWS with remote s3 working directory).

Adding the scripts inside the container, in PATH, insures that the scripts will be always available in the current working directory in the processes, regardless of availability of `scripts/ ` in the cloned repo, that might or might not exist (eg. because only the master node has a copy of the repo)